apps/workbench/config/application.yml.example

```diff
@@ -24,6 +24,9 @@ production:
   secret_token: ~
 
   # You probably also want to point to your API server.
-  arvados_login_base: https://arvados.local:3030/login
-  arvados_v1_base: https://arvados.local:3030/arvados/v1
+  arvados_login_base: https://uuid_prefix.your.domain/login
+  arvados_v1_base: https://uuid_prefix.your.domain/arvados/v1
   arvados_insecure_https: false
+
+  # You might also want to name your workbench
+  site_name: ~
```

Comment:
What's the rationale for adding this? `site_name` already has a reasonable default. (This file should be the smallest set of configs that can get Workbench started -- it doesn't need to mirror `application.default.yml`.)